.firebaserc

```diff
@@ -0,0 +1,5 @@
+{
+  "projects": {
+    "default": "cometplanning"
+  }
+}
```

Comment:
I recommend removing this file, adding it to the `.gitignore` and including instructions in the README on how to set up Firebase.

Reply:
Clarification: I mean keep the file, but remove it from this commit by doing the following:
- Remove the file
- Commit changes
- Update `.gitignore`
- Commit changes
- Move file back